nim/README.md

```diff
@@ -9,13 +9,27 @@ nim compile -d:release --passC:-flto --out:main-nim main.nim
 strip -s main-nim
 ```
 
-For maximum performance ([memory is the trade-off](https://github.com/frol/completely-unscientific-benchmarks/pull/1)):
+For maximum performance ([memory is the trade-off](https://github.com/frol/completely-unscientific-benchmarks/pull/1), currently a bug in Nim 0.18.0, fixed in devel):
 
 ```
 nim compile -d:release --passC:-flto --gc:markAndSweep --out:main-nim main.nim
 strip -s main-nim
 ```
 
+For a faster version that still uses the GC, but breaks the pattern of returning a compound type (like the C++ raw-pointer version):
+
+```
+nim compile -d:release --passC:-flto --gc:markAndSweep --out:main-nim main_fast.nim
+strip -s main-nim
```

Comment:
I think, it is safe to inline this strip by passing `--passL:-s` to `nim compile`.